### PR TITLE
Update organization and tags for xil carma cloud miura release candidate

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description
Update carma config to pull candidate images:
- v2xhub:release-miura
- php:release-miura
- usdotfhwastolcandidate:cdasim:miura
- usdotfhwastolcandidate:carma-cloud:miura

## Related Issue
[CXC-27](https://usdot-carma.atlassian.net/browse/CXC-27)

## Motivation and Context
Verification Testing with release candidate

## How Has This Been Tested?
Locally tested carma config install

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

[CXC-27]: https://usdot-carma.atlassian.net/browse/CXC-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SaikrishnaBairamoni — APPROVED** [review]
Do we need to change Xil_cloud directory? if not it looks good.

**Comment by paulbourelly999** [comment]
> Do we need to change Xil_cloud directory? if not it looks good.

No, this is only for the xil_carma_cloud image.

File: xil_carma_cloud/docker-compose.yml [hunk 1]
```diff
@@ -409,7 +409,7 @@ services:
       - ./mysql:/docker-entrypoint-initdb.d
 
   php:
-    image: usdotfhwaops/php:7.6.0
+    image: ${OPS_DOCKER_ORG:-usdotfhwaops}/php:${OPS_DOCKER_TAG:-develop}
```
Comment (chengyuan0124) [inline xil_carma_cloud/docker-compose.yml:412]: Do we have OPS_DOCKER_ORG in .env?

Comment (paulbourelly999) [reply xil_carma_cloud/docker-compose.yml:412]: No because we use the default value. We could define/override it in the .env file.

**Review by chengyuan0124 — APPROVED** [review]

**Pull request merged**